CICD/ExtensionMethods.cs

```diff
@@ -385,8 +384,8 @@ public static string GetFileVersion(this Project project)
 
         if (string.IsNullOrEmpty(version))
         {
-            // TODO: Create custom exception name MissingFileVersionException 
-            // TODO: In the exception, explain how to set the version 
+            // TODO: Create custom exception name MissingFileVersionException
+            // TODO: In the exception, explain how to set the version
```

Comment:
Do you need these TODO's?
```suggestion
            // TODO: Create custom exception name MissingFileVersionException
            // TODO: In the exception, explain how to set the version
```